FizzBuzz.py

```diff
@@ -1,27 +1,42 @@
-from types import ListType, IntType
+"""Print all integers from 1 to 100. However, for multiples of three,
+print 'Fizz' instead of the number, for multiples of five, print
```

Comment:
I'll have to test to make sure but currently I don't believe you fulfill the requirements. you're supposed to print fizz, buzz or fizzbuzz *instead of* the number. right now you print both